helpers.go

```diff
@@ -278,7 +278,7 @@ func updatePatchCmd(zypperCmd string, ctx *cli.Context) {
 		"replacefiles"}
 	toIgnore := []string{"author", "message"}
 
-	cmd := formatZypperCommand("ref", fmt.Sprintf("-n %v", zypperCmd))
+	cmd := formatZypperCommand("ref", fmt.Sprintf("-n %v", zypperCmd), "clean -a")
```

Comment:
Just I quick question (I don't remember the implementation details of `formatZypperCommand`): is it going to execute everything as a single command, something like:
```
RUN zypper in -n vim && zypper clean -a
```